shared/profile/wot-author/index.tsx

```diff
@@ -75,22 +144,77 @@ export const Question1 = (props: Props) => {
           is the person you think they are?
         </Kb.Text>
       </Kb.Box2>
-      {Constants.choosableWotVerificationTypes.map(vt => (
+      {Constants.choosableWotVerificationTypes.map(vt2 => (
         <VerificationChoice
-          key={vt}
+          key={vt2}
           voucheeUsername={props.voucheeUsername}
-          verificationType={vt}
-          selected={false}
-          onSelect={() => {}}
-        />
+          verificationType={vt2}
+          selected={vt === vt2}
+          onSelect={() => setVt(vt2)}
+        >
+          {vt2 === 'proofs' &&
+            proofs.map(proof => (
+              <Kb.Box2
+                key={`${proof.key}:${proof.value}`}
+                direction="horizontal"
+                alignSelf="stretch"
+                alignItems="center"
+                style={styles.insetContainer}
+              >
+                <Kb.Checkbox
+                  checked={proof.checked && vt === 'proofs'}
+                  onCheck={proof.onCheck}
+                  disabled={vt !== 'proofs'}
+                  labelComponent={
+                    <Kb.Text type="Body">
+                      {proof.value}@{proof.key}
+                    </Kb.Text>
+                  }
+                />
+              </Kb.Box2>
+            ))}
+          {vt2 === 'other' && vt === vt2 && (
```

Comment:
I find this whole chunk hard to read. Maybe move everything inside `<VerificationChoice> ... </VerificationChoice>` into a separate component? I'd be down to discuss if you don't like that option.